DEBUG - 2025-06-14 20:54:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 20:54:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 20:55:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 20:55:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 20:55:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 20:55:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 20:55:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 20:55:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 20:56:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 20:56:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 20:56:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 20:56:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 20:57:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 21:00:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 21:04:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 21:04:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 21:05:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 21:05:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-14 21:05:29 --> ErrorException: Undefined variable $form
[Method: GET, Route: pedidos/new]
in APPPATH\Views\pedidos\form.php on line 4.
 1 APPPATH\Views\pedidos\form.php(4): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $form', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php', 4)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('pedidos/form', [], true)
 5 APPPATH\Controllers\Pedidos.php(22): view('pedidos/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->new()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-14 21:15:48 --> ErrorException: Undefined variable $enderecos
[Method: GET, Route: pedidos/new]
in APPPATH\Views\pedidos\form.php on line 55.
 1 APPPATH\Views\pedidos\form.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $enderecos', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php', 55)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('pedidos/form', [], true)
 5 APPPATH\Controllers\Pedidos.php(25): view('pedidos/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->new()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-14 21:23:11 --> ErrorException: Undefined variable $enderecos
[Method: GET, Route: pedidos/new]
in APPPATH\Views\pedidos\form.php on line 55.
 1 APPPATH\Views\pedidos\form.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $enderecos', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php', 55)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('pedidos/form', [], true)
 5 APPPATH\Controllers\Pedidos.php(25): view('pedidos/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->new()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-14 21:23:12 --> ErrorException: Undefined variable $enderecos
[Method: GET, Route: pedidos/new]
in APPPATH\Views\pedidos\form.php on line 55.
 1 APPPATH\Views\pedidos\form.php(55): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $enderecos', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php', 55)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('pedidos/form', [], true)
 5 APPPATH\Controllers\Pedidos.php(25): view('pedidos/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->new()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-14 21:23:42 --> ErrorException: Undefined property: stdClass::$enderecos_logradouro
[Method: GET, Route: pedidos/new]
in APPPATH\Views\pedidos\form.php on line 57.
 1 APPPATH\Views\pedidos\form.php(57): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$enderecos_logradouro', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php', 57)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('pedidos/form', [], true)
 5 APPPATH\Controllers\Pedidos.php(26): view('pedidos/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->new()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-14 21:24:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 21:26:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-14 21:26:58 --> ErrorException: Undefined variable $clientes
[Method: GET, Route: pedidos/new]
in APPPATH\Views\pedidos\form.php on line 13.
 1 APPPATH\Views\pedidos\form.php(13): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $clientes', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php', 13)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('pedidos/form', [], true)
 5 APPPATH\Controllers\Pedidos.php(30): view('pedidos/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->new()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-14 21:29:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-14 21:29:59 --> ErrorException: Undefined property: stdClass::$clientes_id
[Method: GET, Route: pedidos/new]
in APPPATH\Views\pedidos\form.php on line 14.
 1 APPPATH\Views\pedidos\form.php(14): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$clientes_id', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php', 14)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('pedidos/form', [], true)
 5 APPPATH\Controllers\Pedidos.php(37): view('pedidos/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->new()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-14 21:35:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-14 21:35:15 --> ErrorException: Undefined property: stdClass::$clientes_id
[Method: GET, Route: pedidos/new]
in APPPATH\Views\pedidos\form.php on line 14.
 1 APPPATH\Views\pedidos\form.php(14): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$clientes_id', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php', 14)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('pedidos/form', [], true)
 5 APPPATH\Controllers\Pedidos.php(38): view('pedidos/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->new()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-14 21:38:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-14 21:38:34 --> ErrorException: Undefined property: stdClass::$clientes_id
[Method: GET, Route: pedidos/new]
in APPPATH\Views\pedidos\form.php on line 14.
 1 APPPATH\Views\pedidos\form.php(14): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$clientes_id', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php', 14)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('pedidos/form', [], true)
 5 APPPATH\Controllers\Pedidos.php(38): view('pedidos/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->new()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-14 21:38:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-14 21:38:39 --> ErrorException: Undefined variable $e
[Method: GET, Route: pedidos/new]
in APPPATH\Views\pedidos\form.php on line 14.
 1 APPPATH\Views\pedidos\form.php(14): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $e', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php', 14)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('pedidos/form', [], true)
 5 APPPATH\Controllers\Pedidos.php(38): view('pedidos/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->new()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-14 21:38:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-14 21:38:40 --> ErrorException: Undefined variable $e
[Method: GET, Route: pedidos/new]
in APPPATH\Views\pedidos\form.php on line 14.
 1 APPPATH\Views\pedidos\form.php(14): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $e', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php', 14)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('pedidos/form', [], true)
 5 APPPATH\Controllers\Pedidos.php(38): view('pedidos/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->new()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-14 21:38:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-14 21:38:54 --> ErrorException: Undefined property: stdClass::$clientes_id
[Method: GET, Route: pedidos/new]
in APPPATH\Views\pedidos\form.php on line 14.
 1 APPPATH\Views\pedidos\form.php(14): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$clientes_id', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php', 14)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('pedidos/form', [], true)
 5 APPPATH\Controllers\Pedidos.php(38): view('pedidos/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->new()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-14 21:40:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-14 21:40:12 --> ErrorException: Undefined variable $c
[Method: GET, Route: pedidos/new]
in APPPATH\Views\pedidos\form.php on line 12.
 1 APPPATH\Views\pedidos\form.php(12): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $c', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php', 12)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('pedidos/form', [], true)
 5 APPPATH\Controllers\Pedidos.php(38): view('pedidos/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->new()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-14 21:40:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-14 21:40:22 --> ErrorException: Undefined property: stdClass::$clientes_id
[Method: GET, Route: pedidos/new]
in APPPATH\Views\pedidos\form.php on line 14.
 1 APPPATH\Views\pedidos\form.php(14): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$clientes_id', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php', 14)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('pedidos/form', [], true)
 5 APPPATH\Controllers\Pedidos.php(38): view('pedidos/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->new()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-14 21:48:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-14 21:48:13 --> ErrorException: Undefined property: stdClass::$clientes_id
[Method: GET, Route: pedidos/new]
in APPPATH\Views\pedidos\form.php on line 14.
 1 APPPATH\Views\pedidos\form.php(14): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$clientes_id', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php', 14)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('pedidos/form', [], true)
 5 APPPATH\Controllers\Pedidos.php(38): view('pedidos/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->new()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-14 22:12:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-14 22:12:58 --> ErrorException: Undefined property: stdClass::$clientes_id
[Method: GET, Route: pedidos/new]
in APPPATH\Views\pedidos\form.php on line 14.
 1 APPPATH\Views\pedidos\form.php(14): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$clientes_id', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php', 14)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\pedidos\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('pedidos/form', [], true)
 5 APPPATH\Controllers\Pedidos.php(38): view('pedidos/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->new()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-14 22:14:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 22:17:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 22:18:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 22:19:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 22:20:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 22:20:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 22:20:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 22:20:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 22:20:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 22:20:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-06-14 22:21:12 --> mysqli_sql_exception: Unknown column 'pedidos.produto_id' in 'on clause' in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `pedidos...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `pedidos...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `pedidos...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `pedidos...', Array, false)
#4 C:\xampp\htdocs\codeigniter4\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter4\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter4\app\Controllers\Pedidos.php(17): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Pedidos->index()
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\codeigniter4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2025-06-14 22:21:12 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'pedidos.produto_id' in 'on clause'
[Method: GET, Route: pedidos]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Pedidos.php(17): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-14 22:21:12 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'pedidos.produto_id' in 'on clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Pedidos.php(17): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->index()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-14 22:21:12 --> [Caused by] mysqli_sql_exception: Unknown column 'pedidos.produto_id' in 'on clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Pedidos.php(17): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->index()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
ERROR - 2025-06-14 22:22:30 --> mysqli_sql_exception: Unknown column 'pedidos.produto_id' in 'on clause' in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `pedidos...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `pedidos...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `pedidos...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `pedidos...', Array, false)
#4 C:\xampp\htdocs\codeigniter4\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter4\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter4\app\Controllers\Pedidos.php(17): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Pedidos->index()
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\codeigniter4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2025-06-14 22:22:30 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'pedidos.produto_id' in 'on clause'
[Method: GET, Route: pedidos]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Pedidos.php(17): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-14 22:22:30 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'pedidos.produto_id' in 'on clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Pedidos.php(17): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->index()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-14 22:22:30 --> [Caused by] mysqli_sql_exception: Unknown column 'pedidos.produto_id' in 'on clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Pedidos.php(17): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->index()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
ERROR - 2025-06-14 22:22:51 --> mysqli_sql_exception: Unknown column 'pedidos.produto_id' in 'on clause' in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `pedidos...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `pedidos...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `pedidos...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `pedidos...', Array, false)
#4 C:\xampp\htdocs\codeigniter4\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter4\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter4\app\Controllers\Pedidos.php(17): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Pedidos->index()
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\codeigniter4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2025-06-14 22:22:51 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'pedidos.produto_id' in 'on clause'
[Method: GET, Route: pedidos]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Pedidos.php(17): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-14 22:22:51 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'pedidos.produto_id' in 'on clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Pedidos.php(17): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->index()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-14 22:22:51 --> [Caused by] mysqli_sql_exception: Unknown column 'pedidos.produto_id' in 'on clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Pedidos.php(17): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->index()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-14 22:22:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-06-14 22:22:57 --> mysqli_sql_exception: Unknown column 'pedidos.produto_id' in 'on clause' in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `pedidos...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `pedidos...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `pedidos...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `pedidos...', Array, false)
#4 C:\xampp\htdocs\codeigniter4\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter4\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter4\app\Controllers\Pedidos.php(17): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Pedidos->index()
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\codeigniter4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2025-06-14 22:22:57 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'pedidos.produto_id' in 'on clause'
[Method: GET, Route: pedidos]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Pedidos.php(17): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-14 22:22:57 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'pedidos.produto_id' in 'on clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Pedidos.php(17): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->index()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-14 22:22:57 --> [Caused by] mysqli_sql_exception: Unknown column 'pedidos.produto_id' in 'on clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `pedidos`.*, `produtos`.`produtos_nome`
FROM `pedidos`
JOIN `produtos` ON `produtos`.`produtos_id` = `pedidos`.`produto_id`', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Pedidos.php(17): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Pedidos->index()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pedidos))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-14 22:27:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 22:27:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
WARNING - 2025-06-14 22:27:53 --> [DEPRECATED] password_hash(): Passing null to parameter #1 ($password) of type string is deprecated in APPPATH\Controllers\Clientes.php on line 60.
 1 APPPATH\Controllers\Clientes.php(60): password_hash(Object(SensitiveParameterValue), '2y')
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Clientes->create()
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
ERROR - 2025-06-14 22:27:53 --> mysqli_sql_exception: Column 'usuarios_nome' cannot be null in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codeigniter4\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\codeigniter4\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\codeigniter4\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\codeigniter4\app\Controllers\Clientes.php(51): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Clientes->create()
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
#10 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\codeigniter4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\codeigniter4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2025-06-14 22:27:53 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'usuarios_nome' cannot be null
[Method: POST, Route: clientes/create]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_email`, `usuarios_sobrenome`, `usuarios_cpf`, `usuarios_fone`, `usuarios_nivel`, `usuarios_data_cadastro`, `usuarios_senha`, `usuarios_data_nasc`) VALUES (:usuarios_nome:, :usuarios_email:, :usuarios_sobrenome:, :usuarios_cpf:, :usuarios_fone:, :usuarios_nivel:, :usuarios_data_cadastro:, :usuarios_senha:, :usuarios_data_nasc:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Clientes.php(51): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Clientes->create()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-14 22:27:53 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'usuarios_nome' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_email`, `usuarios_sobrenome`, `usuarios_cpf`, `usuarios_fone`, `usuarios_nivel`, `usuarios_data_cadastro`, `usuarios_senha`, `usuarios_data_nasc`) VALUES (NULL, NULL, NULL, NULL, NULL, 0, \'2025-06-14 22:27:53\', \'$2y$10$cZnKzHILF4x5xBwSxPwqEOAL8dQOMMbwOuQUgqkWhWOel.m5GDAGG\', NULL)')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_email`, `usuarios_sobrenome`, `usuarios_cpf`, `usuarios_fone`, `usuarios_nivel`, `usuarios_data_cadastro`, `usuarios_senha`, `usuarios_data_nasc`) VALUES (NULL, NULL, NULL, NULL, NULL, 0, \'2025-06-14 22:27:53\', \'$2y$10$cZnKzHILF4x5xBwSxPwqEOAL8dQOMMbwOuQUgqkWhWOel.m5GDAGG\', NULL)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_email`, `usuarios_sobrenome`, `usuarios_cpf`, `usuarios_fone`, `usuarios_nivel`, `usuarios_data_cadastro`, `usuarios_senha`, `usuarios_data_nasc`) VALUES (:usuarios_nome:, :usuarios_email:, :usuarios_sobrenome:, :usuarios_cpf:, :usuarios_fone:, :usuarios_nivel:, :usuarios_data_cadastro:, :usuarios_senha:, :usuarios_data_nasc:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Clientes.php(51): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Clientes->create()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-14 22:27:53 --> [Caused by] mysqli_sql_exception: Column 'usuarios_nome' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_email`, `usuarios_sobrenome`, `usuarios_cpf`, `usuarios_fone`, `usuarios_nivel`, `usuarios_data_cadastro`, `usuarios_senha`, `usuarios_data_nasc`) VALUES (NULL, NULL, NULL, NULL, NULL, 0, \'2025-06-14 22:27:53\', \'$2y$10$cZnKzHILF4x5xBwSxPwqEOAL8dQOMMbwOuQUgqkWhWOel.m5GDAGG\', NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_email`, `usuarios_sobrenome`, `usuarios_cpf`, `usuarios_fone`, `usuarios_nivel`, `usuarios_data_cadastro`, `usuarios_senha`, `usuarios_data_nasc`) VALUES (NULL, NULL, NULL, NULL, NULL, 0, \'2025-06-14 22:27:53\', \'$2y$10$cZnKzHILF4x5xBwSxPwqEOAL8dQOMMbwOuQUgqkWhWOel.m5GDAGG\', NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_email`, `usuarios_sobrenome`, `usuarios_cpf`, `usuarios_fone`, `usuarios_nivel`, `usuarios_data_cadastro`, `usuarios_senha`, `usuarios_data_nasc`) VALUES (NULL, NULL, NULL, NULL, NULL, 0, \'2025-06-14 22:27:53\', \'$2y$10$cZnKzHILF4x5xBwSxPwqEOAL8dQOMMbwOuQUgqkWhWOel.m5GDAGG\', NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_email`, `usuarios_sobrenome`, `usuarios_cpf`, `usuarios_fone`, `usuarios_nivel`, `usuarios_data_cadastro`, `usuarios_senha`, `usuarios_data_nasc`) VALUES (:usuarios_nome:, :usuarios_email:, :usuarios_sobrenome:, :usuarios_cpf:, :usuarios_fone:, :usuarios_nivel:, :usuarios_data_cadastro:, :usuarios_senha:, :usuarios_data_nasc:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Clientes.php(51): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Clientes->create()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
WARNING - 2025-06-14 22:28:02 --> [DEPRECATED] password_hash(): Passing null to parameter #1 ($password) of type string is deprecated in APPPATH\Controllers\Clientes.php on line 60.
 1 APPPATH\Controllers\Clientes.php(60): password_hash(Object(SensitiveParameterValue), '2y')
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Clientes->create()
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
ERROR - 2025-06-14 22:28:02 --> mysqli_sql_exception: Column 'usuarios_nome' cannot be null in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codeigniter4\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\codeigniter4\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\codeigniter4\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\codeigniter4\app\Controllers\Clientes.php(51): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Clientes->create()
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
#10 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\codeigniter4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\codeigniter4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2025-06-14 22:28:02 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'usuarios_nome' cannot be null
[Method: POST, Route: clientes/create]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_email`, `usuarios_sobrenome`, `usuarios_cpf`, `usuarios_fone`, `usuarios_nivel`, `usuarios_data_cadastro`, `usuarios_senha`, `usuarios_data_nasc`) VALUES (:usuarios_nome:, :usuarios_email:, :usuarios_sobrenome:, :usuarios_cpf:, :usuarios_fone:, :usuarios_nivel:, :usuarios_data_cadastro:, :usuarios_senha:, :usuarios_data_nasc:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Clientes.php(51): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Clientes->create()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-14 22:28:02 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'usuarios_nome' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_email`, `usuarios_sobrenome`, `usuarios_cpf`, `usuarios_fone`, `usuarios_nivel`, `usuarios_data_cadastro`, `usuarios_senha`, `usuarios_data_nasc`) VALUES (NULL, NULL, NULL, NULL, NULL, 0, \'2025-06-14 22:28:02\', \'$2y$10$IkrcY2syIjhntjtSqXTJsukT8RV5LtEH4tyCGoR0XFZcbIHi9iXC6\', NULL)')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_email`, `usuarios_sobrenome`, `usuarios_cpf`, `usuarios_fone`, `usuarios_nivel`, `usuarios_data_cadastro`, `usuarios_senha`, `usuarios_data_nasc`) VALUES (NULL, NULL, NULL, NULL, NULL, 0, \'2025-06-14 22:28:02\', \'$2y$10$IkrcY2syIjhntjtSqXTJsukT8RV5LtEH4tyCGoR0XFZcbIHi9iXC6\', NULL)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_email`, `usuarios_sobrenome`, `usuarios_cpf`, `usuarios_fone`, `usuarios_nivel`, `usuarios_data_cadastro`, `usuarios_senha`, `usuarios_data_nasc`) VALUES (:usuarios_nome:, :usuarios_email:, :usuarios_sobrenome:, :usuarios_cpf:, :usuarios_fone:, :usuarios_nivel:, :usuarios_data_cadastro:, :usuarios_senha:, :usuarios_data_nasc:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Clientes.php(51): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Clientes->create()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-14 22:28:02 --> [Caused by] mysqli_sql_exception: Column 'usuarios_nome' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_email`, `usuarios_sobrenome`, `usuarios_cpf`, `usuarios_fone`, `usuarios_nivel`, `usuarios_data_cadastro`, `usuarios_senha`, `usuarios_data_nasc`) VALUES (NULL, NULL, NULL, NULL, NULL, 0, \'2025-06-14 22:28:02\', \'$2y$10$IkrcY2syIjhntjtSqXTJsukT8RV5LtEH4tyCGoR0XFZcbIHi9iXC6\', NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_email`, `usuarios_sobrenome`, `usuarios_cpf`, `usuarios_fone`, `usuarios_nivel`, `usuarios_data_cadastro`, `usuarios_senha`, `usuarios_data_nasc`) VALUES (NULL, NULL, NULL, NULL, NULL, 0, \'2025-06-14 22:28:02\', \'$2y$10$IkrcY2syIjhntjtSqXTJsukT8RV5LtEH4tyCGoR0XFZcbIHi9iXC6\', NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_email`, `usuarios_sobrenome`, `usuarios_cpf`, `usuarios_fone`, `usuarios_nivel`, `usuarios_data_cadastro`, `usuarios_senha`, `usuarios_data_nasc`) VALUES (NULL, NULL, NULL, NULL, NULL, 0, \'2025-06-14 22:28:02\', \'$2y$10$IkrcY2syIjhntjtSqXTJsukT8RV5LtEH4tyCGoR0XFZcbIHi9iXC6\', NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_email`, `usuarios_sobrenome`, `usuarios_cpf`, `usuarios_fone`, `usuarios_nivel`, `usuarios_data_cadastro`, `usuarios_senha`, `usuarios_data_nasc`) VALUES (:usuarios_nome:, :usuarios_email:, :usuarios_sobrenome:, :usuarios_cpf:, :usuarios_fone:, :usuarios_nivel:, :usuarios_data_cadastro:, :usuarios_senha:, :usuarios_data_nasc:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Clientes.php(51): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Clientes->create()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-14 22:28:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 22:28:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 22:28:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
WARNING - 2025-06-14 22:28:34 --> [DEPRECATED] password_hash(): Passing null to parameter #1 ($password) of type string is deprecated in APPPATH\Controllers\Clientes.php on line 60.
 1 APPPATH\Controllers\Clientes.php(60): password_hash(Object(SensitiveParameterValue), '2y')
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Clientes->create()
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
ERROR - 2025-06-14 22:28:34 --> mysqli_sql_exception: Column 'usuarios_nome' cannot be null in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codeigniter4\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\codeigniter4\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\codeigniter4\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\codeigniter4\app\Controllers\Clientes.php(51): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Clientes->create()
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
#10 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\codeigniter4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\codeigniter4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2025-06-14 22:28:34 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'usuarios_nome' cannot be null
[Method: POST, Route: clientes/create]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_email`, `usuarios_sobrenome`, `usuarios_cpf`, `usuarios_fone`, `usuarios_nivel`, `usuarios_data_cadastro`, `usuarios_senha`, `usuarios_data_nasc`) VALUES (:usuarios_nome:, :usuarios_email:, :usuarios_sobrenome:, :usuarios_cpf:, :usuarios_fone:, :usuarios_nivel:, :usuarios_data_cadastro:, :usuarios_senha:, :usuarios_data_nasc:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Clientes.php(51): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Clientes->create()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-14 22:28:34 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'usuarios_nome' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_email`, `usuarios_sobrenome`, `usuarios_cpf`, `usuarios_fone`, `usuarios_nivel`, `usuarios_data_cadastro`, `usuarios_senha`, `usuarios_data_nasc`) VALUES (NULL, NULL, NULL, NULL, NULL, 0, \'2025-06-14 22:28:34\', \'$2y$10$8zmY2Mp3HIEiATx/CT9wJ.eDVCFDihCHxEaHXr5L39pAFRqAAi2aK\', NULL)')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_email`, `usuarios_sobrenome`, `usuarios_cpf`, `usuarios_fone`, `usuarios_nivel`, `usuarios_data_cadastro`, `usuarios_senha`, `usuarios_data_nasc`) VALUES (NULL, NULL, NULL, NULL, NULL, 0, \'2025-06-14 22:28:34\', \'$2y$10$8zmY2Mp3HIEiATx/CT9wJ.eDVCFDihCHxEaHXr5L39pAFRqAAi2aK\', NULL)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_email`, `usuarios_sobrenome`, `usuarios_cpf`, `usuarios_fone`, `usuarios_nivel`, `usuarios_data_cadastro`, `usuarios_senha`, `usuarios_data_nasc`) VALUES (:usuarios_nome:, :usuarios_email:, :usuarios_sobrenome:, :usuarios_cpf:, :usuarios_fone:, :usuarios_nivel:, :usuarios_data_cadastro:, :usuarios_senha:, :usuarios_data_nasc:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Clientes.php(51): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Clientes->create()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-14 22:28:34 --> [Caused by] mysqli_sql_exception: Column 'usuarios_nome' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_email`, `usuarios_sobrenome`, `usuarios_cpf`, `usuarios_fone`, `usuarios_nivel`, `usuarios_data_cadastro`, `usuarios_senha`, `usuarios_data_nasc`) VALUES (NULL, NULL, NULL, NULL, NULL, 0, \'2025-06-14 22:28:34\', \'$2y$10$8zmY2Mp3HIEiATx/CT9wJ.eDVCFDihCHxEaHXr5L39pAFRqAAi2aK\', NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_email`, `usuarios_sobrenome`, `usuarios_cpf`, `usuarios_fone`, `usuarios_nivel`, `usuarios_data_cadastro`, `usuarios_senha`, `usuarios_data_nasc`) VALUES (NULL, NULL, NULL, NULL, NULL, 0, \'2025-06-14 22:28:34\', \'$2y$10$8zmY2Mp3HIEiATx/CT9wJ.eDVCFDihCHxEaHXr5L39pAFRqAAi2aK\', NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_email`, `usuarios_sobrenome`, `usuarios_cpf`, `usuarios_fone`, `usuarios_nivel`, `usuarios_data_cadastro`, `usuarios_senha`, `usuarios_data_nasc`) VALUES (NULL, NULL, NULL, NULL, NULL, 0, \'2025-06-14 22:28:34\', \'$2y$10$8zmY2Mp3HIEiATx/CT9wJ.eDVCFDihCHxEaHXr5L39pAFRqAAi2aK\', NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_email`, `usuarios_sobrenome`, `usuarios_cpf`, `usuarios_fone`, `usuarios_nivel`, `usuarios_data_cadastro`, `usuarios_senha`, `usuarios_data_nasc`) VALUES (:usuarios_nome:, :usuarios_email:, :usuarios_sobrenome:, :usuarios_cpf:, :usuarios_fone:, :usuarios_nivel:, :usuarios_data_cadastro:, :usuarios_senha:, :usuarios_data_nasc:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Clientes.php(51): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Clientes->create()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-14 22:33:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 22:37:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 22:39:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 22:43:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 22:43:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 22:43:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 22:50:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 22:50:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 22:51:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 22:52:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 22:58:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 22:58:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 22:59:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-14 23:02:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-14 23:37:38 --> ErrorException: Maximum execution time of 60 seconds exceeded
[Method: GET, Route: pedidos/enderecos_cliente/4]
in SYSTEMPATH\Debug\Exceptions.php on line 208.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
